api-docs/assets-api-introduction.md

```diff
@@ -3,12 +3,33 @@ title: Assets API - Introduction
 category: 63170aaed72d5f00a36a8afe
 ---
 
-Discover the power of our Assets API, your key to manage the asset domain. By leveraging the Assets API, developers can
-easily onboard and offboard assets to Trackunit Iris and get all assets in a fleet as well as update specifications for
-an asset. This API also allows to manage assets across accounts by enabling the share, transfer, unshare and remove
-visibility actions. For more information, see the documentation of the endpoints.
+Discover the power of our Assets API, your key to manage the asset domain. By leveraging the Assets API, developers can easily get all assets in a fleet as well as update specifications for an asset. This API also allows to manage assets across accounts by enabling the share, transfer, unshare and remove visibility actions. For more information, see the documentation of the endpoints.
+
+> 🚧 No onboarding / offboarding support of machine type assets
+> 
+> The Assets API currently does not support onboarding or offboarding assets with the MACHINE asset type. Furthermore, the telematics devices are currently limited to Bluetooth telematics devices. See the [Machine Onboarding API](https://app.swaggerhub.com/apis-docs/trackunit.com/machine-onboarding/1.0.46) for those operations for now.
 
 ## Interface
 
 The API expose an OpenAPI JSON REST interface that you can call directly using your OAuth 2.0 access token.
-See [Access IRIS APIs](../reference/access-token) for further details on how to obtain an access token.
\ No newline at end of file
+See [Access IRIS APIs](../reference/access-token) for further details on how to obtain an access token.
```

Comment:
Do we really need to put this in the intro to all APIs.
Isn't it a given since all APIs use the same pattern